src/apprt/gtk/Surface.zig

```diff
@@ -717,7 +719,12 @@ pub fn redraw(self: *Surface) void {
 }
 
 /// Close this surface.
-pub fn close(self: *Surface, processActive: bool) void {
+pub fn close(self: *Surface, process_active: bool) void {
+    self.closeWithConfirmation(process_active, .{ .surface = self });
+}
```

Comment:
Do we need this function to satisfy an interface or something? It doesn't seem to add much much over the next function.